### PR TITLE
Add custom committer

### PR DESCRIPTION
**Pull request by Lorenzo0111**
## Checks

I have...

- [x] read and understood the [Contributing Guidelines][contributing]
- [x] Updated any nessecary files such as the README.md and/or CHANGELOG.md.

## Type of Pull request

- [x] **Minor Change**  
  This Pull request doesn't break existing configuration.
- [ ] **Major Change**  
  This Pull request will break existing configuration.
- [ ] **Bug fix**  
  This Pull request will fix a (critical) bug.
- [ ] **Documentation**  
  This Pull request only changes documentation (README.md, CHANGELOG.md, etc.)
- [ ] **Other:** `__________` 

## Description

Added an option to edit committer name and email.

Closes #

[contributing]: https://github.com/Readme-Workflows/recent-activity/blob/main/.github/CONTRIBUTING.md

### CODE REVIEW

**Comment by Andre601** [comment]
I for my part can't see the real benefit here. But I let the other maintainers give their feedback

**Comment by PuneetGopinath** [comment]
Hi @Lorenzo0111!
Thanks for this PR.
But currently we this action is in an idle state.
That we announced in our discord server:
```
🥁 Recent Activity action is idle for 2 weeks 🥁

Hello everyone!
We are sad 😢 to inform you that Recent Activity action is idle 😴 for 2 more weeks.
As we also want to give rest to our maintainers and one of our maintainer (Abishek Joshi), his vivas and sems are gonna start from tomorrow (some exams).
We worked continuously for many days to bring new features and fix bugs.
You can still submit feature requests and bug reports, but we will not be able to finish them as our functionality maintainer (Abishek Joshi) is not available, I (Puneet Gopinath) sometimes can fix bugs and fulfill feature requests, but without approval of Abishek we can't merge PRs, etc.
So we mention that Recent Activity action is idle for 2 more weeks.
That means it is idle till 31 May 2021.
```
Please wait for Abishek 🙏.

**Comment by PuneetGopinath** [comment]
Andre, just update the Changelog.

File: src/functions/commitFile.js [hunk 1]
```diff
@@ -14,13 +19,8 @@ const exec = require("./execute");
...
 const commitFile = async () => {
-  await exec("git", [
-    "config",
-    "--global",
-    "user.email",
-    "41898282+github-actions[bot]@users.noreply.github.com",
-  ]);
-  await exec("git", ["config", "--global", "user.name", "readme-bot"]);
+  await exec("git", ["config", "--global", "user.email", commitEmail]);
+  await exec("git", ["config", "--global", "user.name", commitName]);
```
Comment (PuneetGopinath) [inline src/functions/commitFile.js:23]: Where did you define `commitEmail` and `commitName`, aren't they `commit_email` and `commit_name`

**Review by PuneetGopinath — CHANGES_REQUESTED** [review]

**Comment by PuneetGopinath** [comment]
Hi @Lorenzo0111.
Can you give permission for maintaners to edit.
Docs: https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork
After I merge branch main, can you update the docs in `docs/Configuration.md`

**Review by abhijoshi2k — CHANGES_REQUESTED** [review]
Please change the incorrect variable names in `src/functions/commitFile.js` and merge latest commits from `main` branch.
After that, update the `docs/Configuration.md`

**Comment by Lorenzo0111** [comment]
> 
> 
> Hi @Lorenzo0111.
> Can you give permission for maintaners to edit.
> Docs: https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork
> After I merge branch main, can you update the docs in `docs/Configuration.md`

Done
immagine

**Comment by PuneetGopinath** [comment]
Or else you can follow this help: https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/merging-an-upstream-repository-into-your-fork

**Comment by PuneetGopinath** [comment]
@Lorenzo0111 Can you resolve the [requested changes](https://github.com/Readme-Workflows/recent-activity/pull/63#pullrequestreview-669647855)?

**Comment by Lorenzo0111** [comment]
> 
> 
> @Lorenzo0111 Can you resolve the [requested changes](https://github.com/Readme-Workflows/recent-activity/pull/63#pullrequestreview-669647855)?

done

File: dist/index.js [hunk 2]
```diff
@@ -19893,9 +19895,9 @@ const commitFile = async () => {
     "config",
     "--global",
     "user.email",
-    "41898282+github-actions[bot]@users.noreply.github.com",
+    commitEmail,
   ]);
-  await exec("git", ["config", "--global", "user.name", "readme-bot"]);
+  await exec("git", ["config", "--global", "user.name", commitName]);
```
Comment (PuneetGopinath) [inline dist/index.js:19900]: Here also, the same fix is required

**Review by PuneetGopinath — CHANGES_REQUESTED** [review]

File: dist/index.js [hunk 3]
```diff
@@ -19893,9 +19895,9 @@ const commitFile = async () => {
     "config",
     "--global",
     "user.email",
-    "41898282+github-actions[bot]@users.noreply.github.com",
+    commitEmail,
```
Comment (PuneetGopinath) [inline dist/index.js:19898]: This line also

**Review by PuneetGopinath — CHANGES_REQUESTED** [review]

**Review by Andre601 — APPROVED** [review]
You shouldn't request my review here, since I don't have the proper knowledge of this coding language to give proper feedback

**Comment by Lorenzo0111** [comment]
> 
> 
> You shouldn't request my review here, since I don't have the proper knowledge of this coding language to give proper feedback

I didn't requested it. I think it is automatic

**Comment by Andre601** [comment]
> > You shouldn't request my review here, since I don't have the proper knowledge of this coding language to give proper feedback
> 
> I didn't requested it. I think it is automatic

True. Just saw it.

**Comment by PuneetGopinath** [comment]
> You shouldn't request my review here, since I don't have the proper knowledge of this coding language to give proper feedback

He didn't, you are a CODEOWNER

**Comment by PuneetGopinath** [comment]
> > > You shouldn't request my review here, since I don't have the proper knowledge of this coding language to give proper feedback
> > 
> > 
> > I didn't requested it. I think it is automatic
> 
> True. Just saw it.

You have to review it, as their as changes in docs

**Review by PuneetGopinath — APPROVED** [review]

**Review by abhijoshi2k — APPROVED** [review]
It's working properly!

**Pull request merged**